docs/editors/atom.md

```diff
@@ -3,10 +3,7 @@ id: atom
 title: Atom
 ---
 
-The [`ide-scala`](https://atom.io/packages/ide-scala) Atom plugin has
-experimental support for Metals. Please follow the installation instructions of
-the `ide-scala` plugin to setup Metals.
-
-```scala mdoc:requirements
-
-```
+Metals will soon work with Atom thanks to the
```

Comment:
FYI @laughedelic

Reply:
Great, I'll send a PR to update this once it's ready.